examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    generator = partial(inputs_generator, samples=samples)
</pre><p>I don't think this block is needed; it's pretty clear at this point what's going on. It would be more clear to combine the <code>def inputs_generator</code>  block with the <code>tf.data.Dataset.from_generator...</code>  block below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>